rfc#00002

```diff
@@ -0,0 +1,93 @@
+# Title (Ex. 00002: Trust domain introduction)
+- Authors: [Mike Bursell](you@github-email)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-13
+- Status Note: under discussion
+- Start Date: 2020-02-11
+- Tags: trust
+
+## Summary
+
+Trust domains are a key concept within Enarx.  This RFC provides an
+introduction to trust domains.
+
+## Motivation
+
+In order to explain the motivations behind Enarx, our design principles
+and aims, we need to be able to explain what trust domains are.  This RFC
+attempts to provide an introduction.
+
+## Tutorial
+
+A definition of trust, from [What is trust?](http://aliceevebob.com/2017/05/09/what-is-trust-with-apologies-to-pontius-pilate/).
+
+"Trust is the assurance that one entity holds that another will
+perform particular actions according to a specific expectation."
+First corollary: "Trust is always contextual."
+Second corollary: "One of the contexts for trust is always time."
+Third corollary: "Trust relationships are not symmetrical."
+
+### Applying this to Enarx
+The key trust relationship for Enarx is that held by the tenant
+(workload owner) to that workload.  Enarx aims to provide assurances to
```

Comment:
Re-reading this, i think i see why i'm confused by this sentence: "that workload" refers to a workload that hasn't been defined clearly in the main body of the sentence but only referred to within parenthesis. It may be a mistake on my part, but i was taught to look at text in parenthesis is optional in a sentence.
Suggestion: "tenant (workload owner), to that workload" --> "tenant, that is the workload owner, to that workload"

Reply:
clarified.